content.test.ts

```diff
@@ -82,6 +82,12 @@ test(
     expect(content.version).toBe(0);
     expect(content.commands.length).toBe(1);
 
+    // check that prompt that is too large will fail gracefully without hitting the service
+    const hugePrompt = "p".repeat(32 * 1000 * 1000);
+    await expect(() =>
```

Comment:
Do you think this should be a separate test?

Reply:
I considered that, but prompt is a natural fit for testing this because it can be large but also easy to manipulate in test code.